htdocs/core/modules/modTakePos.class.php

```diff
@@ -266,9 +267,45 @@ public function __construct($db)
 	 */
 	public function init($options = '')
 	{
-		global $conf, $db;
+		global $conf, $db, $langs, $user;
+		$langs->load("cashdesk");
 
 		dolibarr_set_const($db, "TAKEPOS_PRINT_METHOD", "browser", 'chaine', 0, '', $conf->entity);
+		
+		//Default customer for Point of sale
+		if (empty(getDolGlobalInt('CASHDESK_ID_THIRDPARTY1'))) {
+			$societe = new Societe($db);
+			$societe->nom = $langs->trans("DefaultPOSThirdLabel");
+			$societe->client = 1;
```

Comment:
You should also set property note_private = 'Default customer automaticaly created by Point Of Sale module activation. Can be used as the default customer in the Point Of Sale setup. Can also be edited or removed.'

Reply:
Approved for me